README.md

```diff
@@ -49,8 +49,11 @@ The easiest way to get up and running is to use the provided docker compose work
 1. Make sure you have [Docker Desktop](https://docs.docker.com/desktop/) installed and running.
 2. Clone your fork `git clone [repository url]`
 3. `cd` into your clone.
-4. Run `docker-compose up --build` to start the app.
+4. Run `docker compose up` to start the app.
 5. Open [http://localhost:3000](http://localhost:3000) and register as a new user.
+6. Run tests: `docker compose run server rails test`
+7. Open the rails console: `docker compose run server rails console`
+8. Run a psql console: `docker compose run server psql`
```

Comment:
When I initially read these I thought it was a typo with "server" in all of them since "rails server" is a command we all know. But then I figured out that "server" is referring to the docker compose line.

I was just asking chatgpt if we can somehow declare a default service within docker compose so that we could leave it out of the run command. It'd be so nice to simply say `docker compose run [arbitrary command]` but it sounds this is not supported.

I was trying to think of a more generic word, maybe "base" or "system" or "command" or "cmd" :) 

`docker compose run cmd rails test`

I admit I'm nitpicking :) I can also just create aliases

Reply:
Yeah for sure, that's part of what I mean about making bin scripts for it. The commands are verbose